prisoners_dilemma.py

```diff
@@ -113,15 +135,13 @@ def play_single_dilemma(player_1: Team,
         if not suppress_exceptions:
             raise
 
-    if player_1_move not in GLOBALS.ACCEPTABLE_RESPONSES or player_1_move == 'n':
+    if player_1_move not in GLOBALS.ACCEPTABLE_RESPONSES:
         player_1_round_score = GLOBALS.CRASH
         player_1_move = 'n'
-        print(f"{player_1.team_name} crashed in a match against {player_2.team_name}, deducting {GLOBALS.CRASH * -1} points as a result")
```

Comment:
relocated this print statement to appear at the end of matchup with the amount of times each player crashed if any in order to prevent spamming the console and to give a better idea of exactly how many crashes there was